Server/package.json

```diff
@@ -31,9 +31,10 @@
     "graphql": "^16.8.1",
     "graphql-rate-limit": "^3.3.0",
     "graphql-shield": "^7.6.5",
+    "graphql-upload": "^16.0.2",
```

Comment:
Do we need these anymore?!

Reply:
nope this was from the earlier attempts.. idk why it keeps popping in.

Reply:
removed it